upgrade_testing/data/upgrade

```diff
@@ -270,12 +282,18 @@ function do_upgrade_and_maybe_reboot() {
     upgrade_log "Attempting to upgrade from ${current} to ${target} (started from ${initial})"
 
     do_normal_upgrade
+    kernel_booted_during_upgrade=$(uname-r)
     exit_with_log_if_nonzero $STATUS "ERROR: Something went wrong with the upgrade."
     maybe_reboot
 
     exit_with_log_if_nonzero $STATUS "ERROR: Something went wrong with the upgrade."
 
     upgrade_log "Upgrading complete."
+    post_upgrade_kernel=$(uname -r)
```

Comment:
No, the kernel after the upgrade will be different, what is important is that the kernel that was running and it's headers, if they were, are still is installed.

So you can `dpkg -l linux-*-$kernel_booted_during_upgrade` or something like that before and after and if that is different fails